www/js/app.js

```diff
@@ -408,13 +409,20 @@ document.getElementById('btnHome').addEventListener('click', function (event) {
 });
 document.getElementById('btnConfigure').addEventListener('click', function (event) {
     event.preventDefault();
+    this.blur();
     // Highlight the selected section in the navbar
     document.getElementById('liHomeNav').setAttribute('class', '');
     document.getElementById('liConfigureNav').setAttribute('class', 'active');
     document.getElementById('liAboutNav').setAttribute('class', '');
     $('.navbar-collapse').collapse('hide');
-    // Show the selected content in the page
-    uiUtil.tabTransitionToSection('config', params.showUIAnimations);
+    if (document.getElementById('configuration').style.display == 'none') {
+        // Show the selected content in the page
+        uiUtil.tabTransitionToSection('config', params.showUIAnimations);
+    }
+    else{
```

Comment:
Please use the standard style rules as required by ESLint. This should be `} else {` (on one line).